platform_channels/lib/src/method_channel_demo.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:platform_channels/src/counter_method_channel.dart';
+
+/// The widget demonstrates how to use [MethodChannel] to invoke platform methods.
+/// It has two [FloatingActionButton]s and a [Text] widget to display the value of
+/// [count].
+class MethodChannelDemo extends StatefulWidget {
+  @override
+  _MethodChannelDemoState createState() => _MethodChannelDemoState();
+}
+
+class _MethodChannelDemoState extends State<MethodChannelDemo> {
+  int count;
+  final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
+
+  @override
+  void initState() {
+    super.initState();
+    count = 0;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      key: _scaffoldKey,
+      appBar: AppBar(
+        title: const Text('MethodChannel Demo'),
+      ),
+      floatingActionButton: Row(
```

Comment:
I believe that if one is going to have two FABs, they should be a in a Column. At least that's the example they show on the Material site:

https://material.io/components/buttons-floating-action-button